src/main/java/io/vertx/core/Vertx.java

```diff
@@ -481,6 +481,33 @@ static void clusteredVertx(VertxOptions options, Handler<AsyncResult<Vertx>> res
   EventLoopGroup nettyEventLoopGroup();
 
   /**
+   * Like {@link #createWorkerExecutor(String, int)} but with the {@link VertxOptions#setWorkerPoolSize} {@code poolSize}.
+   */
+  WorkerExecutor createWorkerExecutor(String name);
+
+  /**
+   * Like {@link #createWorkerExecutor(String, int, long)} but with the {@link VertxOptions#setMaxWorkerExecuteTime} {@code maxExecuteTime}.
+   */
+  WorkerExecutor createWorkerExecutor(String name, int poolSize);
```

Comment:
As the method doesn't always create a new executor, but returns an existing one if it already exists, then perhaps it should be called workerExecutor() (or getOrCreateWorkerExecutor) instead?